src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/ItemPickupListener.java

```diff
@@ -30,6 +34,12 @@ public void onEntityPickup(EntityPickupItemEvent e) {
         else if (e.getItem().getItemStack().hasItemMeta() && e.getItem().getItemStack().getItemMeta().hasDisplayName() && e.getItem().getItemStack().getItemMeta().getDisplayName().startsWith(ITEM_PREFIX)) {
             e.setCancelled(true);
             e.getItem().remove();
+        } else if (SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_16)) {
+            if (e.getEntityType().equals(EntityType.PIGLIN)) {
```

Comment:
Enum comparisons should be done using == instead of .equals() as it allows the JVM to do premature eliminations during compiler-time rather than runtime.

Reply:
Got it